src/Components/InstructionScreenComponents/Instructions.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+function Instructions(){
+    return(
+        <div>
```

Comment:
Instead of a `div`, you could also use an empty `<>...</>` [fragment](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html) tag to wrap this content, which avoids adding an extra node to the DOM.